### PR TITLE
MAAS-1622 | fix issue with trigger event discovering

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: alamo_alerter/alerter.py [hunk 1]
```diff
@@ -47,15 +50,23 @@ def analyze_result(self, results: deque):
...
+            last_result = RESULT_UNKNOWN
+            result_length = len(results)
 
-            if all(res == 1 for res in results) and not last_result:
+            if result_length == results.maxlen:
+                last_result = results.popleft()
+            elif result_length == results.maxlen - 1:
+                last_result = RESULT_OK
+
+            if all([all(result == RESULT_FAILED for result in results),
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: it looks a little bit hard to understand, will anyone remember what it does a few weeks from now?

**Comment by ch40s3k** [comment]
:+1:

File: alamo_alerter/alerter.py (hunk 1 above)
Comment (pnuckowski) [inline alamo_alerter/alerter.py]: @deejay1 comment will help to understand ? :)

**Comment by deejay1** [comment]
:+1:

**Pull request merged**